src/server.jl

```diff
@@ -99,7 +99,10 @@ end
 
 function _check_output_dst(s::AbstractString)
     dir = dirname(s)
-    isdir(dir) || throw(ArgumentError("directory does not exist: $(dir)"))
+    if !isdir(dir)
+        _dir = dirname(joinpath(pwd(), s))
+        isdir(_dir) || throw(ArgumentError("directory does not exist: $(dir)"))
+    end
     return nothing
 end
```

Comment:
```suggestion
function _check_output_dst(s::AbstractString)
    s = abspath(s)
    dir = dirname(s)
    isdir(dir) || throw(ArgumentError("directory does not exist: $(dir)"))
    return nothing
end
```

is probably sufficient.